src/rezplugins/shell/sh.py

```diff
@@ -58,6 +59,13 @@ def get_syspaths(cls):
             if path not in paths:
                 paths.append(path)
         cls.syspaths = [x for x in paths if x]
+
+        # add Rez binaries
+        exe = which("rez")
```

Comment:
This will fail under OSX. OSX has already something called `rez`, see https://www.unix.com/man-page/osx/1/Rez/. I've never been able to use the rez-commands without `-` because of that thing.

Reply:
Thanks @JeanChristopheMorinPerso, fixed.